environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/freeipa/FreeIpaCreationHandler.java

```diff
@@ -125,6 +125,7 @@ public void accept(Event<EnvironmentDto> environmentDtoEvent) {
             }
             eventSender().sendEvent(getNextStepObject(environmentDto), environmentDtoEvent.getHeaders());
         } catch (Exception ex) {
+            LOGGER.error("Error occurred during creating FreeIpa for environment {}.", environmentDto, ex);
```

Comment:
this logger call signature is incorrect, you need to format the message string with String.format